htlcswitch/link_test.go

```diff
@@ -319,7 +320,7 @@ func TestChannelLinkSigThenRev(t *testing.T) {
 
 	const chanAmt = btcutil.SatoshiPerBitcoin * 5
 	const chanReserve = btcutil.SatoshiPerBitcoin * 1
-	aliceLink, bobChannel, batchTicker, start, restore, err :=
+	aliceSwitch, aliceLink, bobChannel, batchTicker, start, restore, err :=
```

Comment:
At this point, perhaps we should morph the harness into an actual struct so we don't have 6+ return values at the call site?

Reply:
Addressed.